chain/indexer/src/lib.rs

```diff
@@ -113,8 +113,8 @@ impl Indexer {
     }
 
     /// Boots up `near_indexer::streamer`, so it monitors the new blocks with chunks, transactions, receipts, and execution outcomes inside. The returned stream handler should be drained and handled on the user side.
-    pub fn streamer(&self) -> mpsc::Receiver<streamer::StreamerMessage> {
-        let (sender, receiver) = mpsc::channel(16);
+    pub fn streamer(&self) -> mpsc::Receiver<StreamerMessage> {
+        let (sender, receiver) = mpsc::channel(100);
```

Comment:
why was it changed to 100? can you add a comment ?

Reply:
In the PR description 

> Increase the streaming channel size from 16 to 100 (reindexing speed)